jupyterlab/tsconfig.json

```diff
@@ -1,7 +1,10 @@
 {
   "compilerOptions": {
-    "target": "ES2020",
-    "moduleResolution": "nodenext",
+    // To ensure compatibility with jupyterlab, options impacting emit (target,module) are copied from
+    // https://github.com/jupyterlab/jupyterlab/blob/c6744e548c7b2790e4c6a4d41a60aae03f22c62a/tsconfigbase.json
+    "target": "ES2018",
+    "module": "esnext",
+    "moduleResolution": "node",
     "outDir": "lib",
```

Comment:
Fixes: `error TS5110: Option 'module' must be set to 'NodeNext' when option 'moduleResolution' is set to 'NodeNext'.`

Setting `module` to `NodeNext` changed how code was emitted, which made me nervous. I decided instead to follow JupyterLab's settings to ensure compatibility.